handler/src/main/java/io/netty/handler/ssl/JdkSslContext.java

```diff
@@ -244,6 +244,8 @@ private SSLEngine configureAndWrapEngine(SSLEngine engine) {
                 case REQUIRE:
                     engine.setNeedClientAuth(true);
                     break;
+                case NONE:
+                    break; // exhaustive cases
```

Comment:
@carl-mastrangelo should we also add:

```java
default: 
    throw Error()
```

This way we will ensure we never miss one ;)